src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsinterventionsservice/dto/DraftReferralDTO.kt

```diff
@@ -53,7 +53,7 @@ data class DraftReferralDTO(
       return DraftReferralDTO(
         id = referral.id,
         createdAt = referral.createdAt,
-        completionDeadline = referral.referralDetails?.completionDeadline,
+        completionDeadline = referral?.completionDeadline,
```

Comment:
Is this what been agreed? are we not going to populate the referralDetails?

Reply:
this isn't necessary